src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -427,5 +529,178 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
             parameterNamesAndLocationsBuilder.Free();
             return (builderAppendCalls.ToImmutableAndFree(), builderPatternExpectsBool ?? false);
         }
+
+        private BoundExpression BindInterpolatedStringHandlerInMemberCall(
```

Comment:
>BindInterpolatedStringHandlerInMemberCall [](http://example.com/codeflow?start=32&length=41)

BindInterpolatedStringHandlerArgument?

Reply:
It's not specific to whether the parameter has the attribute or not, so I prefer the current name.

Reply:
> It's not specific to whether the parameter has the attribute or not, so I prefer the current name.

I don't think I mentioned attribute in this comment.

Reply:
> I don't think I mentioned attribute in this comment.

I ended up inferring it, because `InterpolatedStringHandlerArgumentAttribute` is the name of the attribute.

Reply:
I was referring to method call or indexer access, or quite possibly operator argument.